vic/drivers/image/Makefile

```diff
@@ -247,7 +142,7 @@ tags:	TAGS
 TAGS:	$(SRCS) $(HDRS)
 	etags $(SRCS) $(HDRS)
 clean::
-	\rm -f TAGS
+	rm -f TAGS
```

Comment:
Why is the `\` being removed. Many people have `rm` aliased to `rm -i` or something like it. All that the backslash does is use the unaliased version of `rm`, which is what I think we want here.